client/tests/integration/connected_peers.rs

```diff
@@ -1,15 +1,15 @@
 use std::thread;
 
 use eyre::{Context, Result};
-use iroha_client::{
-    client::Client,
-    data_model::{
-        parameter::{default::MAX_TRANSACTIONS_IN_BLOCK, ParametersBuilder},
-        peer::Peer as DataModelPeer,
-        prelude::*,
-    },
+use iroha_client::{client::Client, data_model::peer::Peer as DataModelPeer};
+use iroha_data_model::{
+    isi::{RegisterExpr, UnregisterExpr},
+    IdBox,
 };
```

Comment:
Why have you changed this part?
Afaik you previously made this change, to have control over `iroha_data_model` export.

Reply:
accidentally, will change back